pkg/pod/ttlDestroyHandler_test.go

```diff
@@ -0,0 +1,59 @@
+package pod
+
+import (
+	"strconv"
+	"testing"
+	"time"
+
+	"github.com/dhis2-sre/rabbitmq/pgk/queue"
+	"github.com/stretchr/testify/mock"
+	"github.com/stretchr/testify/require"
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+type mockQueueProducer struct{ mock.Mock }
```

Comment:
Can you please put the tests first and helpers down.   would stick to this consistently. This way you immediately see tests without having to potentially scroll 😄